pyproject.toml

```diff
@@ -1,31 +1,49 @@
 [tool.poetry]
 name = "radas"
-version = "0.1.0"
-description = "Plasma radiation calculated using OpenADAS"
+# Version is year.month.version
+version = "2024.3.2"
+description = "Plasma radiated power calculated using OpenADAS"
 authors = ["Commonwealth Fusion Systems"]
 readme = "README.md"
+classifiers = [
+"Development Status :: 5 - Production/Stable",
+"Intended Audience :: Science/Research",
+"Programming Language :: Python :: 3",
+"Programming Language :: Python :: 3.9",
+"Programming Language :: Python :: 3.10",
+"Programming Language :: Python :: 3.11",
+"Programming Language :: Python :: 3.12",
+"Programming Language :: Python :: 3 :: Only",
+"Topic :: Scientific/Engineering :: Physics",
+"License :: OSI Approved :: MIT License",
+]
 
 [tool.poetry.scripts]
-radas = 'run_radas:run_radas'
-fetch_adas = 'adas_data.fetch_adas_data:fetch_adas'
+radas = 'radas.cli:run_radas_cli'
+run_radas = 'radas.cli:run_radas_cli'
+radas_config = 'radas.cli:write_config_template'
 
 [tool.poetry.dependencies]
-python = "<3.12,>=3.9"
-numpy = "^1.24.2"
-scipy = "^1.11.2"
-matplotlib = "^3.7.1"
-Pint = "^0.20.1"
+python = "<3.13,>=3.9"
+numpy = "^1.22"
+scipy = "^1.8"
+matplotlib = "^3.6"
+Pint = "^0.21"
 pyyaml = "^6.0"
-xarray = "^2023.3.0"
-pint-xarray = "^0.3"
-click = "^8.1.3"
-
+xarray = ">=2023"
+pint-xarray = ">=0.3"
+click = "^8.1"
+importlib-resources = ">=5.10"
```

Comment:
this is only required for older python version. [`importlib.resources`](https://docs.python.org/3.9/library/importlib.html?highlight=importlib#module-importlib.resources) is part of the standard library

Reply:
Removed this now